.github/workflows/scan_apache2_server.yml

```diff
@@ -2,40 +2,42 @@ name: Scan Apache2 Web Server
 
 on:
   push:
-    branches: [ release ]
+    branches: [release]
   pull_request:
-    branches: [ release ]
+    branches: [release]
 
 jobs:
   build:
-
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.7, 3.8, 3.9]
```

Comment:
Let's not use a test matrix for the scan_xxx_server Actions (it's "too much testing"). Can you revert to Python 3.8 ?